lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -62,9 +63,9 @@ def initialize(aggregate_targets, sandbox, pod_targets, analysis_result, install
         def generate!
           prepare
           install_file_references
-          install_libraries
-          integrate_targets
-          set_target_dependencies
+          @target_installation_results = install_libraries
```

Comment:
does this need to be an ivar?

Reply:
yes unfortunately for this PR. It seems that `#write` method depends on this result and someone to have called `generate!` first. 

I want to clean it up in a follow up PR if its OK.